src/components/spreadsheet/utils/equipment-table-editors.js

```diff
@@ -133,7 +162,7 @@ export const NumericalField = forwardRef(
         function renderNumericText() {
             return (
                 <TextField
-                    value={value}
+                    value={value || ''}
```

Comment:
Not linked to the initial issue but fixes a warning when filling an initially empty field